packages/ui5-middleware-fe-mockserver/README.md

```diff
@@ -77,16 +72,16 @@ Each service must provide at least two things
 On top of that you can specify one of the following option
 
 - mockdataPath : the path to the folder containing the mockdata files
-- generateMockData : whether or not you want to use automatically generated mockdata
+- generateMockData : whether or not you want to use automatically generated mockdata <-- TODO: what happens if i set this to false and only define 1 of 2 entites in the service? error in the frontend?
```

Comment:
no data in the frontend, if you say generateMockData: false you declare that you will take care of all the mockdata yourself

Reply:
That makes sense, but i can define 1 out of 3 OData Services with mockdata and 2 out of 3 are coming with proxy from live data, correct?